static/_redirects

```diff
@@ -21,9 +21,7 @@
 /eventing/* go-get=1 /golang/eventing.html 200
 /eventing-autoscaler-keda/* go-get=1 /golang/eventing-autoscaler-keda.html 200
 /eventing-awssqs/* go-get=1 /golang/eventing-awssqs.html 200
-/eventing-camel/* go-get=1 /golang/eventing-camel.html 200
```

Comment:
cc @n3wscott should automation be dropping these when they're archived?

Reply:
No, they are manually created. But we can document it.